src/interactableComponents/drawer/newDrawer.js

```diff
@@ -118,8 +122,8 @@ export default class NewDrawer extends BaseComponent {
 
   // eslint-disable-next-line react/prop-types
   renderAction = ({item, index, progress, itemsCount}) => {
-    const {itemsTintColor, itemsIconSize, itemsTextStyle} = this.getThemeProps();
-
+    const {itemsTintColor, itemsIconSize, itemsTextStyle, minWidth} = this.getThemeProps();
+    const width = _.max([item.width, minWidth]);
```

Comment:
no need for that, react native's style support the `minWidth` prop. 

Also, If the user doesn't pass a `width` to an item but does pass the `minWidth` prop, the item width will be fixed and wouldn't stretch according to its content.

Reply:
Done